drp.tex

```diff
@@ -162,10 +162,15 @@ \subsubsection{Detection, Deblending and Measurement}
 S\'ersic model \citep{1963BAAA....6...41S, 1968adga.book.....S} fits are run on all available bands simultaneously \cite[MultiProFit,][]{DMTN-312}.
 The resulting S\'ersic model fluxes are provided as an alternative to CModel and are intended to represent total galaxy fluxes.
 Like CModel, the S\'ersic model is a Gaussian mixture approximation to a true S\'ersic profile, convolved with a Gaussian mixture approximation to the \gls{PSF}.
-CModel measurements use a double ``shapelet'' \citep{2003ARA&A..41..645R} PSF with a single shared shape, while the S\'ersic fits use a double Gaussian with independent shape parameters for each component.
 S\'ersic model fits also include a free centroid, with all other structural parameters shared across all bands.
 That is, the intrinsic model has no color gradients, but the convolved model may have color gradients if the \gls{PSF} parameters vary significantly between bands.
 
+CModel measurements use a double ``shapelet'' \citep{2003ARA&A..41..645R} PSF model with a single shared shape.
+The S\'ersic fits are intended to use a double Gaussian with independent shape parameters for each component.
+Due to a pipeline misconfiguration, the S\'ersic fits actually used the shapelet PSF parameters, with the higher-order terms ignored (since MultiProFit does not support shapelet PSFs).
+This bug has since been fixed and is not expected to impact the galaxy fluxes significantly, since the higher-order shapelet PSF parameters tend to be small.
```

Comment:
```suggestion
This bug has since been corrected, and the fix will be applied in future DR campaigns. 
It is not expected to significantly affect the recovered galaxy fluxes in the DP1 data, even though the bug is present, as the higher-order shapelet coefficients are typically small.
```

Reply:
Yes... I'll clarify and make a new PR against PUB-189.